specification/powerbiprivatelinks/resource-manager/Microsoft.PowerBI/stable/2020-06-01/powerbiprivatelinks.json

```diff
@@ -0,0 +1,1627 @@
+            "schema": {
+              "type": "object",
+              "$ref": "#/definitions/TenantResource"
+            },
+            "headers": {}
+          },
+          "default": {
+            "description": "Default Response. It will be deserialized as per the Error definition specified in the schema. Exception will be thrown.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "PowerBIResources"
+        ],
+        "operationId": "PowerBIResources_Delete",
+        "x-ms-examples": {
+          "Deletes private link service resource": {
+            "$ref": "./examples/PowerBIResources_Delete.json"
+          }
+        },
+        "description": "Deletes a Private Link Service Resource for Power BI.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AzureResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "headers": {}
+          },
+          "204": {
+            "description": "No Content."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.PowerBI/privateLinkServicesForPowerBI/{azureResourceName}/privateLinkResources": {
+      "get": {
+        "tags": [
+          "PrivateLinkResources"
+        ],
+        "summary": "List private link Power BI resource.",
+        "description": "List private link resources under a specific Power BI resource.",
+        "operationId": "PrivateLinkResources_ListByResource",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AzureResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "$ref": "#/definitions/PrivateLinkResourcesListResult"
+            },
+            "headers": {}
+          },
+          "default": {
+            "description": "Default Response. It will be deserialized as per the Error definition specified in the schema. Exception will be thrown.",
```

Comment:
>Default Response. It will be deserialized as per the Error definition specified in the schema. Exception will be thrown. [](http://example.com/codeflow?start=28&length=120)

Wording.

Reply:
updated